presto-docs/src/main/sphinx/develop/functions.rst

```diff
@@ -282,3 +282,25 @@ function follows:
   is used when performing a ``GROUP BY`` aggregation, and an implementation
   will be automatically generated for you, if you don't specify a
   ``AccumulatorStateFactory``
+
+Deprecated Function
+-------------------
+
+The ``@Deprecated`` annotation can be used to mark a function as deprecated.
+It should no longer be used. Presto generates a warning whenever a SQL statement
```

Comment:
This is a little ambiguous. It's not clear whether it refers to the function or the annotation. Maybe rephrase as:

> The ``@Deprecated`` annotation can be used to mark a function as deprecated and to indicate that it should no longer be used.

@mosabua, do you have any suggestions?

Reply:
haha.. that was my suggestion .. I thought it is clear as one sentence. 
Maybe we use

The ``@Deprecated`` annotation has to be used on any function that should no longer be used. The annotation causes Presto to generate a warning whenever...